api/jobs.yaml

```diff
@@ -149,6 +149,19 @@ definitions:
         type: string
       display:
         type: string
+      editable:
+        type: boolean
+        default: false
+      fieldType:
+        type: string
+        default: text
+        enum:
+          - text
+          - list
+      fieldValues:
```

Comment:
nit: Could you add a description to this field to make it really clear what it means? I also might name it `possibleValues` or `possibleFieldValues`, fieldValues feels a little vague, but up to you.